### PR TITLE
Update x/sys to v0.0.0-20210630005230-0f9fa26af87c

### PR DESCRIPTION
**Pull request by qbit**
This allows age to be built on OpenBSD/mips64!

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -7,6 +7,8 @@ golang.org/x/net v0.0.0-20190404232315-eb5bcb51f2a3/go.mod h1:t9HGtf8HONx5eT2rtn
 golang.org/x/sys v0.0.0-20190215142949-d0b11bdaac8a/go.mod h1:STP8DvDyc/dI5b8T5hshtkjS+E42TnysNCUPdjciGhY=
 golang.org/x/sys v0.0.0-20191026070338-33540a1f6037 h1:YyJpGZS1sBuBCzLAR1VEpK193GlqGZbnPFnPV/5Rsb4=
```
Comment (FiloSottile) [inline go.sum:8]: I think this is missing a “go mod tidy”?

Comment (qbit) [reply go.sum]: yep, sorry about that! Fixed.

Comment (FiloSottile) [reply go.sum]: Thank you! age running everywhere :D

Comment (qbit) [reply go.sum]: :D \o/

**Pull request merged**